baselines/dom.generated.d.ts

```diff
@@ -3465,6 +3466,9 @@ interface Element extends Node, GlobalEventHandlers, ElementTraversal, NodeSelec
     readonly scrollWidth: number;
     readonly tagName: string;
     innerHTML: string;
+    readonly assignedSlot: HTMLSlotElement | null;
```

Comment:
I do not see why a readonly propoerty can not be undefined. this is useful for users who want to feature dection. this should be:

```ts
readonly assignedSlot?: HTMLSlotElement | null;
```